requirements/requirements.in

```diff
@@ -22,7 +22,7 @@ ghdiff==0.4
 gunicorn
 lxml~=4.3.0
 kafka-python~=1.4
-kombu!=4.3.0
+kombu==4.2.2.post1
```

Comment:
@nickpell I was definitely wrong about how `make upgrade-requirements` would handle this. I've pinned it as you had originally